### PR TITLE
Updating Bluetooth Pairing document

### PR DESCRIPTION
**Pull request by jpwalden**
Updating Bluetooth Pairing document according to the JB#60737

### CODE REVIEW

File: Support/Help_Articles/Bluetooth_Pairing/README.md [hunk 1]
```diff
@@ -157,60 +157,56 @@ You will need to enable the **[developer mode](/Support/Help_Articles/Enabling_D
 ### Install the tools
 
 ```
-\[defaultuser\] devel-su       ## Type your SSH password
+[defaultuser\] devel-su       ## Type your SSH password
```
Comment (jovirkku) [inline Support/Help_Articles/Bluetooth_Pairing/README.md:160]: One more backslash to remove.

File: Support/Help_Articles/Bluetooth_Pairing/README.md [hunk 2]
```diff
@@ -157,60 +157,56 @@ You will need to enable the **[developer mode](/Support/Help_Articles/Enabling_D
...
 
 ### Do the pairing
 
 ```
-\[root\] bluetoothctl
+[root] bluetoothctl
 Agent registered
 
-\[bluetooth\]# scan on
+[bluetooth\]# scan on
```
Comment (jovirkku) [inline Support/Help_Articles/Bluetooth_Pairing/README.md:172]: One more backslash to remove.

File: Support/Help_Articles/Bluetooth_Pairing/README.md [hunk 3]
```diff
@@ -157,60 +157,56 @@ You will need to enable the **[developer mode](/Support/Help_Articles/Enabling_D
...
+[CHG] Device 3C:01:EF:C7:5F:A2 UUIDs: 0000180a-0000-1000-8000-00805f9b34fb
+[CHG] Device 3C:01:EF:C7:5F:A2 ServicesResolved: yes
+[CHG] Device 3C:01:EF:C7:5F:A2 Paired: yes        ## Pairing succeeded
 Pairing successful
-\[CHG\] Device 3C:01:EF:C7:5F:A2 ServicesResolved: no
-\[CHG\] Device 3C:01:EF:C7:5F:A2 Connected: no
-\[bluetooth\]# exit
+[CHG] Device 3C:01:EF:C7:5F:A2 ServicesResolved: no
+[CHG] Device 3C:01:EF:C7:5F:A2 Connected: no
+[bluetooth\]# exit
```
Comment (jovirkku) [inline Support/Help_Articles/Bluetooth_Pairing/README.md:200]: One more backslash to remove.

File: Support/Help_Articles/Bluetooth_Pairing/README.md (hunk 1 above)
Comment (jpwalden) [reply Support/Help_Articles/Bluetooth_Pairing/README.md]: removed

File: Support/Help_Articles/Bluetooth_Pairing/README.md (hunk 2 above)
Comment (jpwalden) [reply Support/Help_Articles/Bluetooth_Pairing/README.md]: removed

File: Support/Help_Articles/Bluetooth_Pairing/README.md (hunk 3 above)
Comment (jpwalden) [reply Support/Help_Articles/Bluetooth_Pairing/README.md]: removed

File: Support/Help_Articles/Bluetooth_Pairing/README.md [hunk 4]
```diff
@@ -157,60 +157,56 @@ You will need to enable the **[developer mode](/Support/Help_Articles/Enabling_D
 ### Install the tools
 
 ```
-\[defaultuser\] devel-su       ## Type your SSH password
+[defaultuser] devel-su       ## Type your SSH password
```
Comment (vigejolla) [inline Support/Help_Articles/Bluetooth_Pairing/README.md:160]: I would also add the prompt characters, `$` for regular user and `#` for root, to these lines. Many are familiar with them, and even those who aren't will understand quickly when they see the prompt on the terminal.

**Review by vigejolla — APPROVED** [review]
LGTM

**Pull request merged**